File: Java/RemoveElement.java
class Solution {
    public int removeElement(int[] nums, int val) {
        
        int count=0;
        
        int i=0,j=0;
        
        while(i<nums.length)
        {
            if(nums[i]!=val)
            {
                count++;
                //int temp=nums[j];
                nums[j]=nums[i];
                //nums[i]=temp;
                j++;
            }
            i++;
        }
       return count;
    }
}

File: Python/RemoveElement.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count = 0
        i=0
        j=0
        
        while i<len(nums):
            if nums[i] != val:
                count+=1
                nums[j]=nums[i]
                j+=1
            i+=1
        
        return count